examples/api/test/material/chip/deletable_chip_attributes.on_deleted.0_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/chip/deletable_chip_attributes.on_deleted.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Chip.onDeleted can be used to delete chips', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.OnDeletedExampleApp(),
+    );
+
+    expect(find.widgetWithText(AppBar, 'DeletableChipAttributes.onDeleted Sample'), findsOne);
+    expect(find.widgetWithText(Chip, 'Aaron Burr'), findsOne);
+    expect(find.widgetWithText(Chip, 'Alexander Hamilton'), findsOne);
+    expect(find.widgetWithText(Chip, 'Eliza Hamilton'), findsOne);
+    expect(find.widgetWithText(Chip, 'James Madison'), findsOne);
+
+    // Delete Alexander Hamilton.
+    await tester.tap(
+      find.descendant(of: find.widgetWithText(Chip, 'Alexander Hamilton'), matching: find.byIcon(Icons.cancel)),
```

Comment:
```suggestion
      find.descendant(
        of: find.widgetWithText(Chip, 'Alexander Hamilton'),
        matching: find.byIcon(Icons.cancel),
      ),
```

Reply:
Can we also put in a method that returns `Finder` and reuse in this test.

Reply:
Sure, is [refactor: Create utility method to find chip delete button](https://github.com/flutter/flutter/pull/152361/commits/fecfa6fbe95092afb65b8f9b2bbdf5275b030eea) fine with you?